app/controllers/remit_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RemitController < ApplicationController
+  def new
+    @event = Event.find_by(id: params[:id])
+    @manager = @event.manager_profile
+    @remit = @event.build_remit
+    if @remit.save
+      @remit
+    else
+      flash[:notice] = "Event cannot yet request a remittance!"
+      redirect_to my_events_path
```

Comment:
it's not just the one-liner part. It's also the current way of writing this. I think this convention is Rails 3. If Rubocop bugs you too much, you can increase the length allowed to `80` 
OR 
break it
```
redirect_to my_events_path,
notice: "Event.....
```